src/mod_last.erl

```diff
@@ -131,7 +131,7 @@ process_local_iq(_From, _To, Acc,
 get_node_uptime() ->
     case ejabberd_config:get_local_option(node_start) of
         {_, _, _} = StartNow ->
-            now_to_seconds(p1_time_compat:timestamp()) - now_to_seconds(StartNow);
+            erlang:system_time(seconds) - now_to_seconds(StartNow);
```

Comment:
I realized just now, that `seconds` is also deprecated time unit :) the correct one is just `second` without the `s`. I'm not going back to all the previous places where `seconds` unit were used, I sure you'll find them easily. Also I stop pointing other deprecated time units unless I spot one I didn't comment yet.